### ISSUE TITLE
testcases.json contains 2 versions of Inapplicable Example 6 for akn7bn: Iframe with interactive elements is not excluded from tab-order

### ISSUE THREAD

**Issue by dd8** [auto-wcag/auto-wcag#2222]
There are two test cases for Inapplicable Example 6 in https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases.json

https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/akn7bn/a13349ef619df6256bd52acb8008760c3711f5dd.html
https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/akn7bn/c88fcaf4d90e2156de75a1cdad8734a3d75c49e4.html

These are identical apart from a semi-colon and extra whitespace in the `<script>` element

testcases.json contains this

```
    {
      "ruleId": "akn7bn",
      "ruleName": "Iframe with interactive elements is not excluded from tab-order",
      "ruleAccessibilityRequirements": {
      ....
      },
      "expected": "inapplicable",
      "testcaseId": "c88fcaf4d90e2156de75a1cdad8734a3d75c49e4",
      "testcaseTitle": "Inapplicable Example 6",
      "relativePath": "testcases/akn7bn/c88fcaf4d90e2156de75a1cdad8734a3d75c49e4.html",
      "url": "https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/akn7bn/c88fcaf4d90e2156de75a1cdad8734a3d75c49e4.html",
      "rulePage": "https://www.w3.org/WAI/standards-guidelines/act/rules/akn7bn/",
      "approved": true
    },
```

and this:

```
    {
      "ruleId": "akn7bn",
      "ruleName": "Iframe with interactive elements is not excluded from tab-order",
      "ruleAccessibilityRequirements": {
     ....
      },
      "expected": "inapplicable",
      "testcaseId": "a13349ef619df6256bd52acb8008760c3711f5dd",
      "testcaseTitle": "Inapplicable Example 6",
      "relativePath": "testcases/akn7bn/a13349ef619df6256bd52acb8008760c3711f5dd.html",
      "url": "https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/akn7bn/a13349ef619df6256bd52acb8008760c3711f5dd.html",
      "rulePage": "https://www.w3.org/WAI/standards-guidelines/act/rules/akn7bn/proposed/"
    },
```

**Edit:** just noticed one of the rulePages shows as https://www.w3.org/WAI/standards-guidelines/act/rules/akn7bn/ and the other as https://www.w3.org/WAI/standards-guidelines/act/rules/akn7bn/proposed/

**Comment by dd8** [comment]
Ok, looks like there are others:

Duplicate test case: Autocomplete attribute has valid value: Inapplicable Example 8
Duplicate test case: Autocomplete attribute has valid value: Inapplicable Example 9
Duplicate test case: Form field has non-empty accessible name: Passed Example 8
Duplicate test case: HTML page has non-empty title: Passed Example 2
Duplicate test case: Iframe with interactive elements is not excluded from tab-order: Inapplicable Example 6
Duplicate test case: Meta viewport allows for zoom: Passed Example 2
Duplicate test case: Meta viewport allows for zoom: Failed Example 2
Duplicate test case: Meta viewport allows for zoom: Failed Example 3
Duplicate test case: Scrollable content can be reached with sequential focus navigation: Passed Example 2
Duplicate test case: Scrollable content can be reached with sequential focus navigation: Inapplicable Example 2
Duplicate test case: Scrollable content can be reached with sequential focus navigation: Inapplicable Example 3
Duplicate test case: Scrollable content can be reached with sequential focus navigation: Inapplicable Example 6
Duplicate test case: Element in sequential focus order has visible focus: Passed Example 4
Duplicate test case: Element in sequential focus order has visible focus: Inapplicable Example 2
Duplicate test case: Text has minimum contrast: Passed Example 7
Duplicate test case: Text has enhanced contrast: Passed Example 6

**Comment by Jym77** [comment]
Yes, as far as I remember, this is for "proposed" test cases that haven't yet been reviewed by TF/AG but have already been merged in ACT rules.

I'll summon @WilcoFiers for the technicalities…
